### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by orikurt**

### CODE REVIEW

File: config/puma.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['RAILS_MAX_THREADS'] || 5)
```
Comment (gaaady) [inline config/puma.rb]: change threads to 1

File: config/puma.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+preload_app!
+
+rackup      DefaultRackup
+port        ENV['PORT']     || 3000
+environment ENV['RACK_ENV'] || 'development'
+
+on_worker_boot do
+  # Worker specific setup for Rails 4.1+
+  # See: https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#on-worker-boot
+  ActiveRecord::Base.establish_connection
```
Comment (gaaady) [inline config/puma.rb]: check what about rails 3.1

File: config/puma.rb (hunk 1 above)
Comment (orikurt) [inline config/puma.rb]: I set RAILS_MAX_THREADS to 1

**Comment by orikurt** [comment]
include basic fb share

**Pull request merged**